src/pages/contributing/component-checklist/index.mdx

```diff
@@ -0,0 +1,157 @@
+| **IBM Figma guidelines**        | Figma components follow the guidelines as defined in [IBM Figma Best Practices](https://www.figma.com/design/4UYvy6eey28bD3IvtNF8xP/IBM-Documentation-Library?node-id=654-4435&t=67lxOP8nwgq7z2E1-0). <br /> <br /> The guidance includes topics on component properties, auto layout, styles, icons, item and base components, content, construction, and file organization. | This checklist ensures components are built correctly before being merged into the main Figma branch.                                                                                                                                                                                                                                                                                               |
+| **IBM Figma Naming Convention** | Figma components follow the component naming convention documented in [IBM Figma Best Practices](https://www.figma.com/design/4UYvy6eey28bD3IvtNF8xP/IBM%C2%AE-Documentation-Library?m=auto&node-id=642-4479&t=1rSjk0nlj8wBV9aK-1).                                                                                                                                           | The naming convention ensures that file architecture and component hierarchies are both standardized and intuitive for the user. <br /> <br /> With this method we are able to have hierarchy between the final components, their bases, and their pieces when we traverse through the assets/instance swapper panels while still being able to view the entire component name when doing a search. |
+| **Built to spec**               | The Figma component should match the [design spec](/contributing/component-checklist/#design-spec) perfectly down to the pixel. Every interaction spec’d should be included in the Figma component.                                                                                                                                                                                        | Thorough implementation with intense attention to detail is pivotal to maintaining pervasive design excellence within the system. This high bar of quality is what sets our system apart and drives adoption, user satisfaction, and overall outcomes.                                                                                                                                              |
```

Comment:
The design spec link here opens an error 404 page.

Reply:
I think this is just broken in the preview because the vercel hasn't updated (I fixed this last week). It should be going to the the design spec section on this page. `/contributing/component-checklist/#design-spec` It looks correct here in the markdown but we'll wait for the deploy to check.

Reply:
Ok these are fixed now.